launch/state_machines/demo_extern.launch

```diff
@@ -27,8 +27,26 @@
             <arg name="config" value="world_modeling/world_model_demo_extern.yaml"/>
         </include>
 
+        <!-- Slack bridge -->
+        <include file="$(find robot_launch_files)/launch/interaction/slack.launch">
+            <arg name="machine" value="sergio2"/>
+            <param name="robot_name" value="$(arg robot_name)" />
+        </include>
+
+        <!-- Telegram bridge -->
+        <include file="$(find robot_launch_files)/launch/interaction/telegram.launch">
+            <arg name="machine" value="sergio2"/>
+            <param name="robot_name" value="$(arg robot_name)" />
+        </include>
+
     </group>
 
+    <!--Start conversation engine-->
+    <node name="conversation_engine" pkg="conversation_engine" type="engine" output="log" respawn="true" args="--robot sergio">
+        <remap from="robot_to_user" to="/sergio/message_to_telegram" />
```

Comment:
If we have both telegram and slack, why use telegram in topics used by both?

Reply:
Good point. it's just what was made in the German Open but should indeed be renamed. Suggestions for a good name?

Reply:
Maybe:
user_to_converstation
converstation_to_user

Reply:
I've named it to ```message_to_user``` and ```message_from_user```